rabbitmq/Dockerfile

```diff
@@ -0,0 +1 @@
+FROM rabbitmq
```

Comment:
Is this all this container needs?

Reply:
Yes

Reply:
It uses some environment variables but those are in the env and docker-compose files